CenterForOpenScience/osf.io
# [ENG-2375] [Hotfix] Tweaks and additional question for RWE Registration Schema

## Purpose
Make some changes to the RWE Registration Schema after it's been further reviewed

## Changes
1. Change the template name/title to "Real World Evidence Recommended Minimum Study Registration Template", old name was 'Real World Evidence in Health Outcomes Minumum Recommended Form'
2. Change q4 (Extraction date) help text to "This refers to either the version number for the database that will be used to implement the study protocol or the date that the database was created."
3. Change the q18 Question-label to "Follow-up definition"
4. Add another question after q19 for Data handling details in sheet.

(Note: qid incrementation on the schema blocks sheet is off by 1 after q6)

## QA Notes
- Verify title is "Real World Evidence Recommended Minimum Study Registration Template"
- Verify help text for q4 is in line with the schema specified in sheet
- Verify q18 question label is in line with the schema specified in sheet
- Verify q20 is in line with 'Data handling' question as specified in sheet

## Documentation
N/A

## Side Effects
N/A

## Ticket
[JIRA Ticket](https://openscience.atlassian.net/browse/ENG-2375)
